include/multipass/sshfs_mount/sshfs_mounts.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2017-2018 Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef MULTIPASS_SSHFSMOUNTS_H
+#define MULTIPASS_SSHFSMOUNTS_H
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+#include <multipass/process.h>
+#include <multipass/qt_delete_later_unique_ptr.h>
+#include <multipass/virtual_machine.h>
```

Comment:
Should be able to forward declare the `VirtualMachine` class since it's a pointer and take out this include here and put it in the .cpp file instead.